invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitySubmission/RecordCommunitySubmissionModal.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { Message } from "semantic-ui-react";
+import { http } from "react-invenio-forms";
+import {
+  CommunitySelectionModalComponent,
+  SubmitReviewModal,
+} from "react-invenio-deposit";
+
+export class RecordCommunitySubmissionModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      confirmationModalOpen: false,
+      selectedCommunity: null,
+      loading: false,
+      error: null,
+    };
+  }
+
+  changeSelectedCommunity = (community) => {
+    this.setState({
+      selectedCommunity: community,
+      error: null,
+    });
+    this.openConfirmModal();
+  };
+
+  canDirectPublish = () => {
+    const { community } = this.state;
+    const { userCommunitiesMemberships } = this.props;
+    const userMembership = userCommunitiesMemberships[community?.id];
+    return userMembership && community?.access.review_policy === "open";
+  };
+
+  handleSuccessSubmit = (data) => {
+    const { handleSuccessAction } = this.props;
+    this.closeConfirmModal();
+    if (this.canDirectPublish()) {
```

Comment:
@kpsherva @jrcastro2 let's do something differently here, this is not reliable.
1. the backend should return:
```
{
    'processed': [
        {
            'community_id': ...,
            'request_id': ...,
            'request': {
                'id': ...,
                'state/status': ...,
                ...
            }
        },
    ],
    'errors': []
}
```
2. the frontend should parse that, and when the request is 'accepted' AND 'closed', then simply show the msg, if instead open, then show the request created msg.